APCAppCore/APCAppCore/Library/Objects/APCConstants.m

```diff
@@ -55,7 +55,9 @@
 NSString *const APCScoringHealthKitDataIsAvailableNotification = @"APCScoringHealthKitDataIsAvailableNotification";
 NSString *const APCTaskResultsProcessedNotification = @"APCTaskResultsProcessedNotification";
 
+NSString *const APCSchedulerUpdatedScheduledTasksNotification = @"APCSchedulerUpdatedScheduledTasksNotification";
```

Comment:
This is fired in the same place as APCUpdateTasksReminderNotification, and not listened for anywhere. Is it actually needed?